src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Verbs.cs

```diff
@@ -43,6 +43,10 @@ internal partial class AddVerbParser
                         "--valid-authentication-types",
                         Strings.SourcesCommandValidAuthenticationTypesDescription,
                         CommandOptionType.SingleValue);
+                    CommandOption protocolVersion = SourceCmd.Option(
+                        "-v|--protocol-version",
```

Comment:
`-v` abbreviation usually reserved verbosity, so in order to avoid confusion let's not use `-v` here.
Technically `-p` is closer to `--protocol-version`, but we don't have to implement shorter abbreviations for option. Just look at `--valid-authentication-types` 3-4 lines above, it doesn't have one.

Reply:
Changed